static/css/CardComponent.scss

```diff
@@ -0,0 +1,87 @@
+@mixin aspect-ratio($width, $height) {
+  position: relative;
+  &:before {
+    display: block;
+    content: "";
+    width: 100%;
+    padding-top: ($height / $width) * 100%;
+  }
+  > .content {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+  }
+}
+
+//10px Gutter
+.gutter-8.row {
```

Comment:
what are these gutter classes doing and why is bootstrap's default behavior not acceptable?

Reply:
The bootstrap gutters were larger than desired so we updated them to be smaller